starter-code/scripts/controllers/articleController.js

```diff
@@ -21,6 +20,7 @@
   };
 
   // COMMENT: What does this method do?  What is it's execution path?
+  //This function is called when a author is selected from the dropdown.  It calles Article.findWhere.  Article.findWhere is executing a sql SELECT useing the author name as the value of the WHERE clause, and passes authorData as the callback. authorData is saving the author name as a context variable. It is part of the middleware chain for /author/:authorName.
```

Comment:
"...authorData is saving the author name as a context variable"
Not quite. It's saving something as a context variable but it's not the author name.